src/main/java/seedu/quotesify/commands/add/AddBookCommand.java

```diff
@@ -0,0 +1,72 @@
+package seedu.quotesify.commands.add;
+
+import seedu.quotesify.author.Author;
+import seedu.quotesify.book.Book;
+import seedu.quotesify.book.BookList;
+import seedu.quotesify.commands.add.AddCommand;
+import seedu.quotesify.commands.Command;
+import seedu.quotesify.exception.QuotesifyException;
+import seedu.quotesify.lists.ListManager;
+import seedu.quotesify.store.Storage;
+import seedu.quotesify.ui.TextUi;
+
+import java.util.logging.Level;
+
+public class AddBookCommand extends AddCommand {
+    public AddBookCommand(String arguments) {
+        super(arguments);
+    }
+
+    public void execute(TextUi ui, Storage storage) {
+        BookList books = (BookList) ListManager.getList(ListManager.BOOK_LIST);
+        addBook(books, ui);
+    }
+
+    private void addBook(BookList books, TextUi ui) {
+        try {
+            String[] titleAndAuthor = information.split(Command.FLAG_AUTHOR, 2);
+            // if user did not provide author, let titleAndAuthor[1] be empty string
+            if (titleAndAuthor.length == 1) {
+                titleAndAuthor = new String[]{titleAndAuthor[0], ""};
+            }
+
+            String title = titleAndAuthor[0].trim();
+            String authorName = titleAndAuthor[1].trim();
+
+            checkMissingInformation(title, authorName);
+            Book newBook = createNewBook(books, title, authorName);
+
+            books.add(newBook);
+            books.sort();
+            ui.printAddBook(newBook);
+
+        } catch (QuotesifyException e) {
+            ui.printErrorMessage(e.getMessage());
+            addLogger.log(Level.INFO, "add book to booklist failed");
+        }
+    }
```

Comment:
Furthermore, should the notations that are pointed at in the attached screenshot about the same sequence diagram an instance of the object? Should they be ":UI", ":AddBookCommand", ...?
Should the arrows that are circled in the attached screenshot be pointed at the start of the activation bar?
![sequence_diagram_notations](https://user-images.githubusercontent.com/55688719/97386665-44898880-190f-11eb-93eb-9de3bfbb9115.PNG)